Gemfile

```diff
@@ -8,7 +8,7 @@ if branch = ENV['TRAVIS_BRANCH']
   same_branch_exists = `curl --head https://github.com/neo4jrb/neo4j-core/tree/#{branch} | head -1`.match(/200 OK/)
   gem 'neo4j-core', github: 'neo4jrb/neo4j-core', branch: same_branch_exists ? branch : 'master'
 else
-  gem 'neo4j-core', github: 'neo4jrb/neo4j-core'
```

Comment:
@cheerfulstoic Ooops, I committed it by mistake ;(

Maybe we can require local path by using an ENV?